2020-02-Spring/4_graphs/README.md

```diff
@@ -48,22 +48,78 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. PROBLEM 2
```

Comment:
@mduw move this to Problem 3; it seems complex enough. I have something simpler in mind and can do Problem 1

Reply:
fixed!